### PR TITLE
JVM config overrides

### PR DESCRIPTION
**Pull request by razvan**
# Description

CI: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hbase-operator-it-custom/105/

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] CRD changes approved
- [x] Helm chart can be installed and deployed operator works
- [x] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: deploy/config-spec/properties.yaml [hunk 1]
```diff
@@ -5,6 +5,125 @@ spec:
...
+        - name: "networkaddress.cache.ttl"
+          kind:
+            type: "file"
+            file: "security.properties"
+      datatype:
+        type: "integer"
+        min: "0"
+      recommendedValues:
+        - fromVersion: "0.0.0"
+          value: "5"
```
Comment (sbernauer) [inline deploy/config-spec/properties.yaml:42]: Nit: Maybe we can increase this number a bit

File: docs/modules/hbase/pages/usage-guide/overrides.adoc [hunk 2]
```diff
@@ -50,6 +51,33 @@ All override property values must be strings. The properties will be formatted a
 
 For a full list of configuration options we refer to the HBase https://hbase.apache.org/book.html#config.files[Configuration Documentation].
 
+=== The security.properties file
+
+The `security.properties` file is used to configure JVM security properties. It is very seldom that users need to tweak any of these, but there is one use-case that stands out, and that users need to be aware of: the JVM DNS cache.
+
+The JVM manages it's own cache of successfully resolved host names as well as a cache of host names that cannot be resolved. Some products of the Stackable platform are very sensible to the contents of these caches and their performance is heavily affected by them. As of version 3.4.12, Apache Hbase performs poorly if the positive cache is disabled. To cache resolved host names, and thus speeding up Hbase queries you can configure the TTL of entries in the positive cache like this:
```
Comment (sbernauer) [inline docs/modules/hbase/pages/usage-guide/overrides.adoc:58]: Please also mention what the default values are, .e.g by using them in the snippet below

**Review by sbernauer — APPROVED** [review]

File: docs/modules/hbase/pages/usage-guide/overrides.adoc (hunk 2 above)
Comment (razvan) [reply docs/modules/hbase/pages/usage-guide/overrides.adoc:58]: Done.

File: deploy/config-spec/properties.yaml (hunk 1 above)
Comment (razvan) [reply deploy/config-spec/properties.yaml]: Raised to 10 secs

**Pull request merged**